File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/spring/model/request/EventoDTO.java
package org.tfg.inhometfgcarloshernandez.spring.model.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EventoDTO {
    private String tipo;
    private String nombre;
    private String votacion;
    private String horaComienzo;
    private String horaFin;
    private String organizador;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/EventoDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model

import com.example.inhometfgandroidcarloshernandez.common.Constantes

data class EventoDTO (
    val id : Int = 0,
    val tipo: String = Constantes.NADA,
    val nombre: String=Constantes.NADA,
    val votacion: String=Constantes.NADA,
    val horaComienzo: String=Constantes.NADA,
    val horaFin: String=Constantes.NADA,
    val organizador: String =Constantes.NADA,
)